src/Components/LandingPage/LandingPage.js

```diff
@@ -32,122 +37,149 @@ export const LandingPage = () => {
-      </PageHeader>
-      <section className="pf-l-page__main-section pf-c-page__main-section">
-        {!isBeta() && showBetaAlert && (
-          <Alert
-            className="pf-u-mb-xl"
-            isInline
-            variant="default"
-            title="Try new features in our Preview environment."
-            actionClose={
-              <AlertActionCloseButton onClose={() => setShowBetaAlert(false)} />
-            }
-            actionLinks={
-              <Button
-                isInline
-                component="a"
-                variant="link"
-                href="/preview/insights/image-builder/landing"
-              >
-                Enter Preview environment
-              </Button>
+          <p>
+            Launch Amazon Web Services or Microsoft Azure hosts to the cloud
+            from the console.
+          </p>
+          <p>
+            Link custom repositories and build any supported image with custom
+            content.
+          </p>
+        </Alert>
+      )}
+      {isBeta() && (
+        <ExpandableSection
+          className="pf-m-light pf-u-mb-xl expand-section"
+          toggleText="Help get started with Preview features"
+          onToggle={setShowHint}
+          isExpanded={showHint}
+          displaySize="large"
+        >
+          <p className="pf-u-pb-sm">
+            For help getting started, access the quick starts for our Preview
+            features.
+          </p>
+          <p className="pf-u-pt-sm">
+            <Button
+              icon={<ArrowRightIcon />}
+              iconPosition="right"
+              variant="link"
+              isInline
+              component="a"
+              onClick={activateQuickstart('insights-launch-aws')}
+              className="pf-u-font-weight-bold"
+            >
+              Launch an AWS Image
+            </Button>
+          </p>
+          <p className="pf-u-pt-sm">
+            <Button
+              icon={<ArrowRightIcon />}
+              iconPosition="right"
+              variant="link"
+              isInline
+              component="a"
+              onClick={activateQuickstart('insights-launch-azure')}
+              className="pf-u-font-weight-bold"
+            >
+              Launch an Azure Image
+            </Button>
+          </p>
+          <p className="pf-u-pt-sm">
+            <Button
+              icon={<ArrowRightIcon />}
+              iconPosition="right"
+              variant="link"
+              isInline
+              component="a"
+              onClick={activateQuickstart('insights-custom-repos')}
+              className="pf-u-font-weight-bold"
+            >
+              Build an Image with Custom Content
+            </Button>
+          </p>
+        </ExpandableSection>
+      )}
+      <ImagesTable />
+    </section>
+  );
+  return (
+    <Suspense fallback="loading...">
```

Comment:
I think I would prefer to wrap only the imported module (specifically `<EdgeImagesTable>`) in `<Suspense >` for now... 

Just a bit of context on why: we aren't using `<Suspense>` in this project (apart from the other imported module, the Launch wizard). For cases where we need to load data, we are currently using RTK Query hooks and conditional rendering. RTK Query does not support `<Suspense>` at the moment because, according to the React docs, "Suspense-enabled data fetching without the use of an opinionated framework is not yet supported".

Reply:
thank you for your explanation:)